commands/import.go

```diff
@@ -83,76 +88,84 @@ func (i *Importer) parseArgs() error {
 		return cli.NewExitError(err.Error(), -1)
 	}
 
-	// check if rolling flag was passed in
-	if i.rolling {
-
-		// verify that required flag values were provided
-		if (i.totalChunks == -1) || (i.currentChunk == -1) {
-			return cli.NewExitError("\n\t[!] Both `--numchunks <total number of chunks>` and `--chunk <current chunk number>` must be provided for rolling analysis import.", -1)
-		}
-
-		// verifies the chunk is a divisor of 24 (we currently support 24 hour's worth of data in a rolling dataset)
-		if !(i.totalChunks > 0) || ((24 % i.totalChunks) != 0) {
-			return cli.NewExitError("\n\t[!] Total number of chunks must be a divisor of 24 (Valid chunk sizes: 1, 2, 4, 6, 8, 12, 24)", -1)
-		}
-
-		// validate chunk size
-		if !(i.currentChunk > 0) {
-			return cli.NewExitError("\n\t[!] Current chunk number must be greater than 0", -1)
-		}
-
-		if i.currentChunk > i.totalChunks {
-			return cli.NewExitError("\n\t[!] Current chunk number cannot be greater than the total number of chunks", -1)
-		}
-
-	}
-
 	i.res = resources.InitResources(i.configFile)
 
 	return nil
 }
 
-func (i *Importer) setTargetDatabase() error {
-	i.res.DB.SelectDB(i.targetDatabase)
+func checkFilesExist(files []string) error {
+	for _, file := range files {
+		if !util.Exists(file) {
+			return cli.NewExitError(fmt.Errorf("\n\t[!] %v cannot be found", file), -1)
+		}
+	}
 	return nil
 }
 
-func (i *Importer) checkImportDirExists() error {
+// setRolling determines what the current rolling and chunk settings should be
+// based on the values already in the database and the values supplied on the
+// command line by the user.
+func setRolling(dbExists bool, dbIsRolling bool, dbCurrChunk int, dbTotalChunks int,
```

Comment:
This is the function that makes all the rolling arguments optional. It's a bunch of if-statements that make assumptions based on the data we have. The comments should explain well enough what it's doing and the unit tests should give a fair confirmation that it's working as it should.